conf/map/battle/skill.conf

```diff
@@ -330,3 +330,10 @@ bowling_bash_area: 0
 // punch a hole into SG it will for example create a "suck in" effect.
 // If you disable this setting, the knockback direction will be completely random (eAthena style).
 stormgust_knockback: true
+
+// Config for full protect skill creator.
```

Comment:
Need to explain better what this config does.

Reply:
@EyesOfAHawk On the last commit, I tried to explain better.